build.xml

```diff
@@ -204,94 +220,34 @@
 	
   <target name="test" depends="compile" description="run tests">
     <echo message="starting ${package} tests"/>
-    <junit failureproperty="test.failed" fork="yes">
+    <junit failureproperty="test.failed" fork="no" printsummary="on">
```

Comment:
Fork should be yes. This is a workaround for a bug with some versions of ant that causes the tests to fail.